pages/common/swc.md

```diff
@@ -0,0 +1,28 @@
+# swc
+
+> JavaScript and TypeScript compiler written in Rust.
+> More information: <https://swc.rs>.
+
+- Transpile a specified input file and output to stdout:
+
+`swc {{path/to/file}}`
+
+- Transpile a specified input file and output to a specific file:
+
+`swc {{path/to/input_file}} --out-file {{path/to/output_file}}`
+
+- Transpile the input file every time it is changed:
+
+`swc {{path/to/file}} --watch`
```

Comment:
How about moving this example to be the second one?

Reply:
I followed the `babel ` page, but yes I can't move this.

Reply:
I think it would make more sense as a sequence: this example is more similar to the first example than the custom output file one, and moving it allows the custom output file and the custom output directory examples to be next to each other.